.github/CONTRIBUTING.md

```diff
@@ -1,65 +1,55 @@
-* Make sure your PR is up-to-date with the rest of the code. Often, a simple click on "Update branch" will do the job,
-  but if not, you are asked to rebase the dev branch manually and resolve the problems on your own. You can find help [on the wiki](https://github.com/TeamNewPipe/NewPipe/wiki/How-to-merge-a-PR). That will make the
+* Make sure your PR is up-to-date with the rest of the code. Often, a simple click on "Update branch" will do the job, but if not, you are asked to rebase the dev branch manually and resolve the problems on your own. You can find help [on the wiki](https://github.com/TeamNewPipe/NewPipe/wiki/How-to-merge-a-PR). That makes the
   maintainers' jobs way easier.
-* Please show intention to maintain your features and code after you contributed it. Unmaintained code is a hassle for
-  the core developers, and just adds work. If you do not intend to maintain features you contributed, please think again
-  about submission, or clearly state that in the description of your PR.
-* Respond yourselves if someone requests changes or otherwise raises issues about your PRs.
-* Send PR that only cover one specific issue/solution/bug. Do not send PRs that are huge and consists of multiple
-  independent solutions.
+* Please show intention to maintain your features and code after you contribute a PR. Unmaintained code is a hassle for core developers. If you do not intend to maintain features you plan to contribute, please rethink your submission, or clearly state that in the PR description.
+* Respond if someone requests changes or otherwise raises issues about your PRs.
+* Send PRs that only cover one specific issue/solution/bug. Do not send PRs that are huge and consist of multiple independent solutions.
+* Try to figure out yourself why builds on our CI fail.
 
 ## Communication
 
-* There is an IRC channel on Freenode which is regularly visited by the core team and other developers:
-  [#newpipe](irc:irc.freenode.net/newpipe). [Click here for Webchat](https://webchat.freenode.net/?channels=newpipe)!
-* If you want to get in touch with the core team or one of our other contributors you can send an email to
+* The [#newpipe](irc:irc.freenode.net/newpipe) channel on freenode has the core team and other developers in it. [Click here for webchat](https://webchat.freenode.net/?channels=newpipe)!
+* You can also use a Matrix account to join the Newpipe channel at [#freenode_#newpipe:matrix.org](https://matrix.to/#/#freenode_#newpipe:matrix.org).
+* If you want to get in touch with the core team or one of our other contributors you can send an e-mail to
   <a href="mailto:tnp@newpipe.schabi.org">tnp@newpipe.schabi.org</a>. Please do not send issue reports, they will be ignored and remain unanswered! Use the GitHub issue
-  tracker described above!
-* Feel free to post suggestions, changes, ideas etc. on GitHub or IRC!
+  tracker described above, unless you are using the built-in crash reporter.
```

Comment:
52-54 can be made one line.

Reply:
@opusforlife2 Remove newline where? The line numbers changed.

Reply:
"If you want to get in touch" until "built-in crash reporter"
There are deleted lines in the diff, so it's not letting me select and edit them.

Reply:
Sorted

Reply:
You merged 2 lines. There were 3.